roles/ci_gen_kustomize_values/tasks/olm_values.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: Assert we have needed parameters
+  ansible.builtin.assert:
+    that:
+      - cifmw_ci_gen_kustomize_values_ooi_image is defined
```

Comment:
Does this mean, that if the new config-map grows we will blow up just because this one is missing? :(

Reply:
yeah, i'll probably remove this.